app/views/user_email_mailer/results_email.html.erb

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Email letter</title>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <style>
+      * {
+          margin: 0;
+          padding: 0;
+          box-sizing: border-box;
+          font-family: 'Josefin Sans', 'Arial', 'Ubuntu',  sans-serif;
+          font-style: normal;
+          font-weight:700;
+          text-align: center;
+          line-height: 0;
+      }
+      .board{
+          min-width:1080px;
+      }
+      .lnk-unsubscribe{
+          text-align: left;
+      }
+  </style>
+  <%= stylesheet_link_tag 'email_page' , media: 'all' %>
+</head>
+<body>
+<div class="board mx-auto">
+  <div class="first-row">
+    <div class="mx-auto my-0 envelope">
+      <p class="mx-auto my-0 logo">Logo/Brand</p>
+    </div>
+  </div>
+  <div class="second-row">
+    <div class="calendar-days ml-55">
+      <div class="date"><%= @view_calendar_days.gsub("\n", "<br>").html_safe %></div>
```

Comment:
why we need to change `\n` to `<br>`? it's your method that used in the one place you can rewrite it to use `<br>` originally
for example 
`@view_calendar_days = "#{time_period.start_date.strftime('%d %b.')} <br> – <br>#{time_period.end_date.strftime('%d %b.')}".html_safe`